doc/install/dependencies.rst

```diff
@@ -79,45 +79,33 @@ Redhat linux users can do likewise with:
 MOAB installation
 ~~~~~~~~~~~~~~~~~
 
-As of DAGMC version 3.1, MOAB version 5.1.0 or higher is required. The following
+As of DAGMC version 3.2, MOAB version 5.3.0 or higher is required. The following
```

Comment:
Why is this again? My recollection is that we could still build the latest DAGMC with MOAB v5.1.0.

Reply:
Good question... it may be <5.3.0 ? I'll check...